src/Containers/SearchCards/SearchCards.jsx

```diff
@@ -16,17 +21,41 @@ const Cards = () => {
     needleType,
   } = useParams();
 
+
   useEffect(() => {
+    setLoading(true);
     axios
       .get(
         `https://api.magicthegathering.io/v1/cards?&Page-Size=1&name=${needleName}&power=${needlePower}&toughness=${needleToughness}&type=${needleType}&colors=${needleColor}&rarity=${needleRarity}&set=${needleSet}`
       )
       .then(({ data }) => {
         setMagicCards(data.cards);
+
+        setLoading();
```

Comment:
For consistency's sake, write `setLoading(false)` instead

Reply:
problem fixed